jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/listeners/ResponseFileSaver.java

```diff
@@ -37,9 +40,19 @@ public ResponseFileSaver(String fileNamePrefix) {
   protected TestElement buildTestElement() {
     ResultSaver ret = new ResultSaver();
     ret.setFilename(fileNamePrefix);
+    ret.setSkipAutoNumber(skipAutoNumber);
+    ret.setSkipSuffix(skipSuffix);
     return ret;
   }
 
+  public void setSkipAutoNumber(boolean skipAutoNumber) {
```

Comment:
Can you please add javadoc to these new methods? Additionally, can you please change the name to match same convention as other similar "property" methods by removing the set prefix?

Reply:
Done, thanks for the review. I've also updated the setters to return the ResponseFileSaver object based off of what I've seen in other classes.